apps/drain-safe/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "drain-safe",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gnosis.pm/safe-apps-react-sdk": "2.2.0",
+    "@gnosis.pm/safe-react-components": "^0.3.0",
```

Comment:
This makes me remember that maybe we should upgrade the template to safe-react-components v0.5.0 that it's the latest one. And maybe we should release v0.6.0 shortly, there are many improvements since then